jobs/blog_validation_jobs.py

```diff
@@ -0,0 +1,159 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Validation Jobs for blog models"""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.platform import models
+from jobs import base_jobs
+from jobs import blog_validation_errors
+from jobs.io import ndb_io
+
+import apache_beam as beam
+(blog_models) = models.Registry.import_models([models.NAMES.blog, ])
```

Comment:
Missing comma, which is important:

```suggestion
(blog_models,) = models.Registry.import_models([models.NAMES.blog, ])
```

Reply:
Done.